src/olympia/activity/admin.py

```diff
@@ -31,4 +32,52 @@ def has_delete_permission(self, request, obj=None):
         return False
 
 
+class ReviewActionReasonLogAdmin(admin.ModelAdmin):
+    date_hierarchy = 'created'
+    fields = (
+        'created',
+        'activity_log',
+        'user_email',
+        'reason',
+    )
+    list_display = (
+        'created',
+        'activity_log',
+        'reason',
+        'user_email',
+    )
+    list_filter = ('reason',)
+    readonly_fields = (
+        'created',
+        'activity_log',
+        'user_email',
+    )
+    search_fields = ('activity_log__user__email',)
+    view_on_site = False
+
+    @admin.display()
+    def user_email(self, obj):
+        return obj.activity_log.user.email
+
+    def has_add_permission(self, request):
+        return False
+
+    def has_delete_permission(self, request, obj=None):
+        return False
+
+    def formfield_for_foreignkey(self, db_field, request, **kwargs):
+        if db_field.name == 'reason':
+            kwargs['queryset'] = ReviewActionReason.objects.filter(
+                is_active__exact=True
+            )
+        return super().formfield_for_foreignkey(db_field, request, **kwargs)
+
+    def get_form(self, request, obj=None, **kwargs):
+        form = super(ReviewActionReasonLogAdmin, self).get_form(request, obj, **kwargs)
+        form.base_fields['reason'].widget.can_add_related = False
+        form.base_fields['reason'].widget.can_change_related = False
```

Comment:
what's this for?

Reply:
Without them you get a little pencil icon and a plus-sign icon beside the select for `reason`, which allow you to edit the `reason` and add a new `reason`, and I don't think those functions are required from this screen, and might in fact be confusing to users. This is what it looks without these:
![Screen Shot 2021-11-10 at 12 00 08 PM](https://user-images.githubusercontent.com/142755/141158403-09373873-debf-48ae-854e-1d8bba490cf8.png)

Reply:
Ah. I think there's a different way to do this (we don't set those properties elsewhere) but if this works then 👍